### ISSUE TITLE
Improve map interface

### ISSUE THREAD

**Issue by jazzchatfield** [GenderedIntelligence/the-trans-dimension#217]
## User need

The 2 tone pink maps aren't that easy to read. Came up as #85

## Implementation Notes and thoughts

- Maps in nonstandard colours can be hard to visually process — allow people to change.
- Pink and purple hard to parse.
- Multicoloured is preferable.
- Could use leaflet layers & allow people to choose
- NOT DONE: https://github.com/GenderedIntelligence/the-trans-dimension/issues/238
- NOT DONE: https://github.com/GenderedIntelligence/the-trans-dimension/issues/242

As a compromise:

- [x] #264

# Was Replace static images with dynamic maps

## Description

Would need to use Mapbox in conjunction with a maps package / plugin of some kind. Not sure if Elm has anything or how easy it is to use node packages within!

## Plan

- [ ] Choose a package or plugin

**Comment by katjam** [comment]
@studiosquid 

Shall we...
1) add to the bottom of work queue, 2) close as a won't fix, 3) leave open for future niceness but not important now.

**Comment by studiosquid** [comment]
@katjam I think I need to know what the feedback came back with as mentioned May 2022?

Sounds like the maps have been an ongoing problem but I don't know if the time / resources exists to address them. That they're non clickable and so zoomed in does make them feel pretty pointless to me (they're also not well implemented in terms of fitting the design properly).

I would guess they're a 3 unless we feel they're actively causing issues — if they are aggravating I wonder if we want to bin them and link to an external map service.
